PublicationsRouter/v2/sword-out/REPOSITORY.md

```diff
@@ -4,19 +4,44 @@ This document provides information on how to get your repository set up to take
 
 ## Account Setup
 
-First you will need a Router account, and to configure it for SWORDv2 integration.  See your account page in the 
-router for more information.  You need to provide the following:
-
-1. A username and password for a user in your repository who has the rights to create content via SWORDv2
-2. A collection URL into which the system will deposit the content
-3. Your preferred packaging format for zip files being deposited.  By default (and the only option in this first version) this will be http://purl.org/net/sword/package/SimpleZip
+First you will need a Router account, and set up the SWORDv2 credentials in the 'Manage connection settings' section on your account.  You need to provide the following:
+
+* The repository URL
+* Select a Repository Configuration ([Eprints Native](#eprints), [Eprints RIOXX](#eprints-rioxx-plugin), [DSpace native](#dspaceother))
+* Select an Eprints deposit location if required (Manage Deposit, Review Queue)
+* A username and password for a user in your repository who has the rights to create content via SWORDv2.
+   * For Eprints deposits into *Manage Deposits* a User role is required.
+   * For Eprints deposits into *Review* queue an Editor or Admin role is required.
+   * For DSpace deposits into a collection an Admin role on the collection is required.
+* A collection URL into which the system will deposit the content
+   * Eprints collection: http://eprints.domain.ac.uk/id/contents
+   * DSpace collection: `http://dspace.domain.ac.uk/swordv2/collection/<handle-id>/<collection-id>` (usually obtained from the collection URL `http://dspace.domain.ac.uk/handle/<handle-id>/<collection-id>` find out more: [DSpace URLs and Identifiers](https://wiki.duraspace.org/display/DSDOC5x/Functional+Overview#FunctionalOverview-PersistentURLsandIdentifiers))
+* Your preferred packaging format for zip files being deposited. [more info](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/Packaging.md#what-package-formats-are-supported)
```

Comment:
Change to: `* Enter the URI of your preferred packaging format for zip files being deposited ([more info](https://github.com/sherpaservices/Public-Documentation/blob/master/PublicationsRouter/v2/api/Packaging.md#what-package-formats-are-supported)).   If your repository can accept multiple formats, then enter URIs separated by comma, with most preferred format first.`

So that it appears like this: 
"Enter the URI of your preferred packaging format for zip files being deposited (more info).  If your repository can accept multiple formats, then enter URIs separated by comma, with most preferred format first."